ospd_openvas/db.py

```diff
@@ -79,11 +79,14 @@ def get_database_address(cls) -> Optional[str]:
         return cls._db_address
 
     @classmethod
-    def create_context(cls, dbnum: Optional[int] = 0) -> RedisCtx:
+    def create_context(
+        cls, dbnum: Optional[int] = 0, encoding: Optional[str] = 'latin-1'
+    ) -> RedisCtx:
         """ Connect to redis to the given database or to the default db 0 .
 
         Arguments:
             dbnum: The db number to connect to.
+            encoding: The enconding to be used to read and write.
```

Comment:
```suggestion
            encoding: The encoding to be used to read and write.
```
Sorry, missed this one on the first go.